newIDE/app/src/InAppTutorial/InAppTutorialOrchestrator.js

```diff
@@ -102,6 +102,10 @@ const translateAndInterpolateText = ({
   } else {
     translatedText = i18n._(text.messageDescriptor, data);
   }
+
+  // Something went wrong with the translation, let's hide the text.
+  if (typeof translatedText !== 'string') return undefined;
```

Comment:
a few crashes happen when trying to call `interpolateText` with a translatedText that doesn't have a method `.match(xxx)` (which should exist for every string)
Possible that the translation `i18n._()` doesn't return what's expected?
Instead of crashing, just return nothing.

Reply:
Should it be `''` instead?

Reply:
I believe it will be handled the same way afterwards (not displaying the text)